app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -172,12 +172,13 @@ def has_thumbnail? document
   ##
   # Render the thumbnail, if available, for a document and
   # link it to the document record.
-  # 
+  #
   # @param [SolrDocument]
   # @param [Hash] options to pass to the image tag
   # @param [Hash] url options to pass to #link_to_document
   # @return [String]
   def render_thumbnail_tag document, image_options = {}, url_options = {}
+    return "" unless has_thumbnail?(document) # do not render any html if there is no thumbnail
```

Comment:
interesting, the issue i was trying to fix was in the gallery view, which comes from https://github.com/projectblacklight/blacklight-gallery 

so..this issue is currently not in the other search result views (which is explained by that method).  

So I think we have an option to do something very similar in the gallery gem (check for has_thumbnail?) before rendering. 

The one downside is that we ever decide to actually render something like a placeholder, we may actually want to do it in the the "render_thumbnail_tag" method itself and *remove* the check for has_thumbnail? in https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/_thumbnail_default.html.erb#L1